.idea/enhancements.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="Go" enabled="true" />
```

Comment:
Looks like you accidentally checked in these files.

Reply:
removed.